### PR TITLE
I have solved the #234 issue in this way.

### PR DESCRIPTION
**Pull request by shinesoftware**
I have added in the UserController a protected variable and I have created a setter and a getter method. I have edited the UserControllerFactory.php catching the service and I have set it up in the UserController instance.

 If it is not good, please show me a solution.

### CODE REVIEW

File: src/ScnSocialAuth/Controller/UserController.php [hunk 1]
```diff
@@ -26,6 +26,12 @@ class UserController extends AbstractActionController
      */
     protected $options;
 
+    /**
+     * @var zfcmoduleoptions
+     */
+    protected $zfcmoduleoptions;
+
+
```
Comment (SocalNick) [inline src/ScnSocialAuth/Controller/UserController.php]: Extra white space failing the Travis build

File: src/ScnSocialAuth/Controller/UserController.php [hunk 2]
```diff
@@ -261,4 +267,21 @@ public function getOptions()
 
         return $this->options;
     }
+
+
```
Comment (SocalNick) [inline src/ScnSocialAuth/Controller/UserController.php]: Here too

**Comment by SocalNick** [comment]
Thanks! This seems reasonable, I'll review in more detail if you can fix those whites pace issues that are failing the Travis build.

**Comment by shinesoftware** [comment]
I have deleted the white spaces.

**Comment by shinesoftware** [comment]
Sorry but I don't understand what I have to do ...

**Comment by SocalNick** [comment]
Maybe try running php-can-fixer locally? 

'''
php php-cs-fixer.phar fix -v --dry-run .
'''

I think if you leave off the --dry-run it might just fix it for you.

**Comment by SocalNick** [comment]
I just fixed for you here: https://github.com/SocalNick/ScnSocialAuth/commit/a319745372de6036a28308c1d1fe763f2a523102

Closing

**Pull request closed without merging**